client/package.json

```diff
@@ -62,7 +63,7 @@
     "sanitize.css": "12.0.1",
     "serve": "12.0.0",
     "shelljs": "0.8.4",
-    "socket.io-client": "4.4.1",
+    "socket.io-client": "4.0.0",
```

Comment:
downgraded because of `Failed to parse source map error` related to `base-64 array-buffer`